teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -1340,6 +1340,10 @@ private class BuildImpl(bean: BuildBean,
         instance.service.cancelBuild(id.stringId, request)
     }
 
+    override fun getResultingParameters(): List<Parameter> {
+        return instance.service.resultingProperties(id.stringId).property?.map { ParameterImpl(it) } ?: emptyList()
```

Comment:
Not sure about emptyList here, may be non-null assertion will be better here.

Reply:
Yes, I suppose resulting parameters should always contains at least some predefined values.